src/web_main.py

```diff
@@ -197,4 +197,15 @@ def pull_down_queue_member(queue_id, action):
     return "", 204
 
 
+@app.route("/api/queues/<int:queue_id>/rename-queue", methods=["PUT"])
```

Comment:
Let's change the last part of this route to `name`, so as to keep more in line with the REST style